src-env (mixdbvendors)/_MicroserviceTemplate_.WebApi.IntegrationTests/Config.cs

```diff
@@ -0,0 +1,120 @@
+using System.Net.Http.Headers;
+#if SQLSERVER
+using _MicroserviceTemplate_.EF.SqlServer;
+#else
+using _MicroserviceTemplate_.EF.PostgreSql;
+#endif
+
+namespace _MicroserviceTemplate_.WebApi.IntegrationTests
+{
+    [SetUpFixture]
+    public class Config
+    {
+        private static IConfiguration _configuration;
+
+        public static _MicroserviceTemplate_DbContext DBContext;
+        public static HttpClient HttpClient;
+
+        [OneTimeSetUp]
+        public static void SetUpFixture()
+        {
+            // Load EnvironmentVariables.json
+            var launchSettingsPath = Path.Combine(Directory.GetCurrentDirectory(), "Properties", "EnvironmentVariables.json");
+
+            if (File.Exists(launchSettingsPath))
+            {
+                var jsonContent = File.ReadAllText(launchSettingsPath);
+                var jsonReader = new JsonTextReader(new StringReader(jsonContent));
+                var launchSettings = JObject.Load(jsonReader);
+
+                var environmentVariables = launchSettings.Root;
+                if (environmentVariables != null)
+                {
+                    foreach (JProperty variable in environmentVariables)
+                    {
+                        Environment.SetEnvironmentVariable(variable.Name, variable.Value.ToString());
+                    }
+                }
+            }
+
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("Properties/EnvironmentVariables.json", optional: true, reloadOnChange: true)
+                .AddEnvironmentVariables()
+                .Add(new CustomEnvironmentVariableConfigurationSource())
+                .Build();
+
+            DBContext = CreateDbContext();
+
+            var server = new TestServer(new WebHostBuilder()
+                .UseConfiguration(_configuration)
+                .UseStartup<StartupStub>());
+
+            HttpClient = server.CreateClient();
+
+            var token = _configuration.GetValue<string>("AccessToken");
+            HttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+        }
+
+        public static _MicroserviceTemplate_DbContext CreateDbContext()
+        {
+            var builder = new DbContextOptionsBuilder<_MicroserviceTemplate_DbContext>().EnableSensitiveDataLogging();
+            
+            if (IsCheckConnectionStringExists())
+            {
+                string connectionString = _configuration.GetValue<string>("ConnectionStrings");
+                #if SQLSERVER
+                if (IsInjectDbCredentialsToConnectionString())
+                {
+                    connectionString +=
+                        $";User Id={_configuration.GetValue<string>("AspNetCoreDbUserName")};Password='{_configuration.GetValue<string>("AspNetCoreDbPassword")}'";
+                }
+
+                builder.UseSqlServer(connectionString); 
```

Comment:
@fuadagazade this is the only different line, so the #if def should be only here

Reply:
In the SQL Server ConnectionString contains User Id, in PostgreSQL this is Username